.github/codecov.yml

```diff
@@ -12,6 +12,15 @@ comment:
 coverage:
   round: down
   precision: 2
+  status:
+    project:
+      default:
+        target: auto
+        threshold: 0.5%
+    patch:
+      default:
+        target: auto
+        threshold: 1%
```

Comment:
while i think this one could potentially be increased, because I think it works per file, and sometimes, adding a unique line may be seen as 5% change.